### DISCUSSION TITLE
Pregunta sobre cómo mostrar el estado de las heladeras

### DISCUSSION THREAD

**Discussion by ParedesMicaela** [dds-jv/foro#427] [TPA - 2024] [answered]
Buenas noches, estaba leyendo el enunciado en la parte donde me piden:
```
El mapa de heladeras deberá ser capaz de mostrar, junto a cada heladera su nivel de llenado así como si
necesita atención
```
mi pregunta es si lo que me están pidiendo es mostrar por pantalla estilo printf o si me piden hacer una lista. Por ahí leí que podia usar un hashmap o capaz también podría crear una clase con esos atributos que me piden y ponerlos todos en una lista, no sé cómo piden mostrarlo ni qué conviene. Gracias!

**Comment by flbulgarelli** [comment, marked as answer]
:wave: ¡Hola! 

En estas primeras entregas no estaremos atacando ninguna cuestión relativa a presentación, sino que trabajaremos exclusivamente sobre el modelo. Por lo tanto, no deberías intentar imprimir por pantalla o mostrar por ningún otro medio la información, sino representar en términos de objetos los elementos que se solicitan (por ejemplo, el mapa). 

En este caso, probablemente no es necesario representar al mapa como una clase en sí, dado que no tendrá ningún comportamiento particular por ahora, pero sí como una lista de puntos, o simplemente como una lista de heladeras, las cuales necesitan incluir información de coordenadas geográficas, nivel de llenado, etc.
